client/js/lounge.js

```diff
@@ -168,6 +168,10 @@ $(function() {
 		});
 	}
 
+	if (navigator.platform.match(/(Mac|iPhone|iPod|iPad)/i)) {
+		$(document.body).addClass("is-osx");
```

Comment:
Technically speaking, there is no OSX terminology anymore so `is-apple` or `is-apple-os` or `is-apple-product` would be more accurate, and maybe more consistent / better for codebase search but up to you, no big deal.

Reply:
Is there such a thing as <kbd>Cmd</kbd> on iP{hone|od|ad)? I thought the keyboard shortcuts were only usable on "desktop" devices.

Reply:
Bluetooth keyboards.